### ISSUE TITLE
 Optimise code in admin_auth_activity.xml

### ISSUE THREAD

**Issue by MohamedMedhat1998** [oppia/oppia-android#1959]
This issue is tracking the optimization work of `admin_auth_activity.xml`

1. Make sure that the layout uses `ConstraintLayout` and there is minimal nesting of views. If this view already uses `ConstraintLayout` then it's fine.

2. Make sure that all TextViews use the style from `styles.xml` file like `Heading1`, `TextFieldLabel`, `Body`, etc. whichever suits best.

While making the above changes the UI should not get affected and the end result should still be the same as before (or an optimized version of UI).

If there are multiple files with the same name, update them all.

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

**Comment by bladebunny** [comment]
I submitted a PR that flattens the view hierarchy and uses standard (Heading2 and Subtitle1) text styles.  I also made some small adjustments to the margins to be more standard.  Landscape view was using a scrollview which did not appear necessary which I removed.  I was not able to test completely because I wasn't sure how to get to the view through app usage.  But I did verify the layouts for different sized devices in the layout editor.

See: https://github.com/oppia/oppia-android/pull/1963
